plugins/module_utils/foreman_helper.py

```diff
@@ -131,6 +131,38 @@ def _patch_location_api(self):
         _location_update_params_location = next(x for x in _location_update['params'] if x['name'] == 'location')
         _location_update_params_location['params'].append(_location_organizations_parameter)
 
+    def _patch_sync_plan_api(self):
+        """This is a workaround for the broken sync_plan apidoc in katello.
```

Comment:
where is the Katello issue link? :)

Reply:
https://projects.theforeman.org/issues/27532